website/api/zkvm/optimization.md

```diff
@@ -513,7 +509,6 @@ below.
 [`env::read_slice`]: https://docs.rs/risc0-zkvm/*/risc0_zkvm/guest/env/fn.read_slice.html
 [example-ecdsa]: https://github.com/risc0/risc0/tree/main/examples/ecdsa
 [example-waldo]: https://github.com/risc0/risc0/tree/main/examples/waldo
-[Executor]: https://github.com/risc0/risc0/blob/main/risc0/zkvm/src/host/server/exec/monitor.rs#L30-L39
```

Comment:
does lychee not like this link?

Reply:
The code was refactored when parallel preflight landed and so there isn't a clear single place in the code that we can point to that refers to the same thing as in the past.

Reply:
kk makes sense -- that confirms my instinct that this was a "stale link" thing rather than a "lychee is opinionated" thing